crates/subspace-node/src/commands/run.rs

```diff
@@ -115,6 +116,12 @@ pub async fn run(run_options: RunOptions) -> Result<(), Error> {
     info!("🏷  Node name: {}", subspace_configuration.network.node_name);
     info!("💾 Node path: {}", base_path.display());
 
+    if maybe_domain_configuration.is_some() && subspace_configuration.sync == ChainSyncMode::Snap {
```

Comment:
I'd probaby use `sync != Full` though

Reply:
The current condition is consistent with the error message. Not critical though. Please, let me know if you still want a change along with the different error message.

Reply:
It is fine, I'm just thinking about invatiants. Fundamental expectation here is not that Snap sync is not supported, but that only full sync is supported instead. The fact that the only other option is snap sync is a coincidence.

In other words if we add a third sync mechanism tomorrow this code will break and if we flip the condition to check for full sync instead it will not break.

Reply:
It makes perfect sense. However, the error message should also be changed if we make this alteration. Something like "domains support only full-sync" which is slightly less obvious IMO than the current one.